bin/tape

```diff
@@ -18,11 +18,11 @@ if (typeof opts.require === 'string') {
 }
 
 opts.require.forEach(function(module) {
-    if (module) {
-      /* This check ensures we ignore `-r ""`, trailing `-r`, or
-       * other silly things the user might (inadvertently) be doing. */
-      require(resolveModule(module, { basedir: cwd }));
-    }
+    var options  = { basedir: cwd, extensions: Object.keys(require.extensions) };
+        
+    /* This check ensures we ignore `-r ""`, trailing `-r`, or
+     * other silly things the user might (inadvertently) be doing. */
+    module && require(resolveModule(module, options));
```

Comment:
Please revert back to the `if (module)` form - abusing value selection operators for control flow isn't a good style.

Reply:
@ljharb certainly! Can you please provide recent documentation of this stance? As a library author and someone who works at the spec level I can almost guarantee that because this may not be the olde hat style. That hat is still old. However I do have strong opinions loosely held. Would really appreciate a link of sorts to update my thinking. 

That said, correcting change now no problem at all for the sake of the repo style. 👍

Reply:
Corrected in 278c8b569a6bd4da6ca9cf71626be2ac843b7f91

Reply:
FWIW @ljharb I agree this may not be the style of the repo. However "abuse" is a harsh word and in modern ecmascript styles is a very constricting mindset. Coincidentally am currently working with google team to begin converting parts of code in said patterns. https://github.com/webcomponents/custom-elements/pull/102#issuecomment-331528755

To be clear from _User Land_ best practices this is 💯  correct. However as library authors who are often restricted to payload bytesize this is actually a practice that shaves many many bytes off the final bundle. Just a thought I was having. I still concur that pattern is orthogonal to the styles in this repo. Whatever you need from me let me know.

Reply:
Best practices require a very constricting mindset, and library authors should not be concerned with payload size; that's a top-level app concern. A minifier is capable of reducing patterns like this just fine without needing to reduce the quality of the code.

I'd refer you to https://github.com/airbnb/javascript for good documentation of JS styles that tend to largely represent the majority of the community.

Thanks for making the change!